### PR TITLE
fix: make dao-local-deploy.sh work again

### PR DESCRIPTION
**Pull request by sanbir**

### CODE REVIEW

**Review by arwer13 — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -158,7 +158,7 @@ __metadata:
     fs-extra: ^8.1.0
     ganache-core: ^2.10.1
     http-proxy: ^1.18.0
-    ipfs-http-client: ^42.0.0
+    ipfs-http-client: 43.0.0
```
Comment (TheDZhon) [inline yarn.lock:161]: Please add a brief explanation regarding the strict `ipfs-http-client` version pinning :pray:

**Review by TheDZhon — CHANGES_REQUESTED** [review]
LGTM, except only one minor point, thank you for the PR :+1:

File: yarn.lock (hunk 1 above)
Comment (sanbir) [reply yarn.lock:161]: Added https://github.com/lidofinance/lido-dao/pull/436/commits/0f64bf1b0df6a2bb5ba2335718453baad9feab71

**Review by TheDZhon — APPROVED** [review]
We are heading to the release :+1:

**Review by arwer13 — APPROVED** [review]

**Pull request merged**